app/models/generic_object_definition.rb

```diff
@@ -6,7 +6,11 @@ class GenericObjectDefinition < ApplicationRecord
   has_one   :picture, :dependent => :destroy, :as => :resource
   has_many  :generic_objects
 
-  def defined_attributes
-    @defined_attributes ||= properties[:attributes].stringify_keys
+  def defined_property_attributes
+    @defined_property_attributes ||= properties[:attributes].stringify_keys
```

Comment:
True. Currently the keys are not normalized. I plan to have another PR to always save them as string type; then we don't need to cache them.